selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/git/GitStatusBar.java

```diff
@@ -59,57 +60,41 @@ public GitStatusBar(SeleniumWebDriver seleniumWebDriver) {
   @FindBy(xpath = Locators.GIT_BAR_HIDE_BTN)
   WebElement gitBarHideBtn;
 
-  /** wait appears git statusbar tab */
-  public void waitGitStatusBarOpenedTab() {
-    new WebDriverWait(seleniumWebDriver, LOAD_PAGE_TIMEOUT_SEC)
-        .until(ExpectedConditions.visibilityOf(statusBarTab));
-  }
-
   /** wait appears git status Bar information panel */
   public void waitGitStatusBarInfoPanel() {
-    new WebDriverWait(seleniumWebDriver, 10)
-        .until(ExpectedConditions.visibilityOf(gitStatusBarInfoPanel));
+    seleniumWebDriverHelper.waitVisibility(gitStatusBarInfoPanel);
   }
 
   public String getTextStatus() {
-    return gitStatusBarInfoPanel.getText();
+    return seleniumWebDriverHelper.waitVisibilityAndGetText(gitStatusBarInfoPanel);
   }
 
   public void waitCloseGitStatusBarInfoPanel() {
-    new WebDriverWait(seleniumWebDriver, LOAD_PAGE_TIMEOUT_SEC)
-        .until(ExpectedConditions.invisibilityOfElementLocated(By.id(Locators.GIT_BAR_INFO_PANEL)));
-  }
-
-  /** click on statusbar tab */
-  public void clickOnGitStatusBarTab() {
-    waitGitStatusBarOpenedTab();
-    statusBarTab.click();
+    seleniumWebDriverHelper.waitInvisibility(By.id(Locators.GIT_BAR_INFO_PANEL));
   }
 
   public void waitMessageInGitTab(final String message) {
     waitGitStatusBarInfoPanel();
-    new WebDriverWait(seleniumWebDriver, LOAD_PAGE_TIMEOUT_SEC)
-        .until((ExpectedCondition<Boolean>) webDriver -> getTextStatus().contains(message));
+
+    seleniumWebDriverHelper.waitNoExceptions(
+        () ->
+            seleniumWebDriverHelper.waitSuccessCondition(
+                webDriver -> getTextStatus().contains(message), LOAD_PAGE_TIMEOUT_SEC),
+        StaleElementReferenceException.class);
   }
 
   /** wait and click on navigate button in the 'git console' */
   public void waitAndClickOnNavigateBtn() {
-    new WebDriverWait(seleniumWebDriver, LOAD_PAGE_TIMEOUT_SEC)
-        .until(ExpectedConditions.elementToBeClickable(gitStatusBarNaviBtn));
-    gitStatusBarNaviBtn.click();
+    seleniumWebDriverHelper.waitAndClick(gitStatusBarNaviBtn);
   }
 
   /** wait and click on remove button in the 'git console' */
   public void waitAndClickOnRemoveBtn() {
-    new WebDriverWait(seleniumWebDriver, LOAD_PAGE_TIMEOUT_SEC)
-        .until(ExpectedConditions.elementToBeClickable(gitStatusBarRemoveBtn));
-    gitStatusBarRemoveBtn.click();
+    seleniumWebDriverHelper.waitAndClick(gitStatusBarRemoveBtn);
   }
 
   /** click on the 'minimize 'button */
   public void clickOnStatusBarMinimizeBtn() {
```

Comment:
What about renaming `clickOnStatusBarMinimizeBtn` to `waitAndClickOnStatusBarMinimizeBtn`?

Reply:
It seems, this name is more exactly. OK, I'll change it.